frontend/app/src/components/settings/api-keys/ServiceKey.vue

```diff
@@ -33,14 +33,13 @@ const cancellable = ref<boolean>(false);
 const errorMessages = useRefMap(status, (status) => {
   if (!status || status.success)
     return [];
-
+  set(editMode, true);
```

Comment:
what is it doing here?

Reply:
I don't think it's safe to change value of ref inside computed.
Move it to watch(status) instead